File: JavaScript/1020-IdadeEmDias.js
var input = require('fs').readFileSync('stdin', 'utf8');
var line = input.split('\n');
var day = parseInt(line.shift());
var ano = day/365;
var mes = (day%365)/30;
var dia = (day%365)%30;
console.log(`${Math.floor(ano.toFixed(1))} ano(s)`);
console.log(`${Math.floor(mes.toFixed(1))} mes(es)`);
console.log(`${Math.floor(dia.toFixed(1))} dia(s)`);

File: Java/LinguagemC/1020 - IdadeEmDias.cpp
#include <stdio.h>
int main() {
    int dias, ano, mes, dia;

    scanf("%i",&dias);

    ano=dias/365;
    mes=(dias/365)/30;
    dia=(dias%365)%30;

    printf("%d ano(s)\n",ano);
    printf("%d mes(es)\n",mes);
    printf("%d dia(s)\n",dia);
    return 0;
}